www/utils/markdown.ts

```diff
@@ -1,6 +1,6 @@
-export * as gfm from "https://deno.land/x/gfm@0.1.19/mod.ts";
-import "https://esm.sh/prismjs@1.27.0/components/prism-jsx.js?no-check";
-import "https://esm.sh/prismjs@1.27.0/components/prism-typescript.js?no-check";
-import "https://esm.sh/prismjs@1.27.0/components/prism-tsx.js?no-check";
+export * as gfm from "https://deno.land/x/gfm@0.1.23/mod.ts";
+import "https://esm.sh/prismjs@1.29.0/components/prism-jsx.js?no-check";
+import "https://esm.sh/prismjs@1.29.0/components/prism-typescript.js?no-check";
+import "https://esm.sh/prismjs@1.29.0/components/prism-tsx.js?no-check";
```

Comment:
```suggestion
import "https://esm.sh/prismjs@1.27.0/components/prism-jsx.js?no-check";
import "https://esm.sh/prismjs@1.27.0/components/prism-typescript.js?no-check";
import "https://esm.sh/prismjs@1.27.0/components/prism-tsx.js?no-check";
```

These versions need to stay in sync with the version in x/gfm.